src/components/Tabs/tabs.module.scss

```diff
@@ -0,0 +1,33 @@
+$light-grey: #d4d4d5;
+
+.heading {
+  text-align: left;
+}
+
+.tabContainerClassName {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  
+  .tabButton {
+    background: white;
+    border: none;
+    padding: 14px 9px 9px;
+    font-size: 0.8em;
+    border-radius: 3px 3px 0 0;
+    cursor: pointer;
+  }
+  .active {
+    font-weight: bold;
+    border: 1px solid $light-grey;
+    background: white;
+  }
+}
+
+.open {
+  opacity: 1;
+}
+
+.closed {
```

Comment:
Aren't `open` and `closed` used for mounting and unmounting a component?
If yes, then using the opacity property is not suitable.
Set it to its default display type.